tests/kafkatest/services/security/minikdc.py

```diff
@@ -48,6 +49,16 @@ def start_node(self, node):
         self.logger.info("minikdc.properties")
         self.logger.info(props_file)
 
+        # create local temp dir where we will store keytab and krb5conf files
```

Comment:
This local file stuff is all pretty much due to the fact that ducktape doesn't have a convenient mechanism to scp between worker nodes

I think it'll be a lot cleaner to add a utility method in `tests/kafkatest/utils/remoteaccount.py` to  encapsulate this behavior for now

Something like:
```
def scp(source_node, source_path, target_node, target_path):
  # source_node.account.copy_from(...) source_path -> local temp file
  # target_node.account.copy_to(...) local temp file -> target_path
```

Then you shouldn't have to track all of these "local" paths